### ISSUE TITLE
Vulkan Validation Layers don't work in Proton with pressure-vessel enabled

### ISSUE THREAD

**Issue by DadSchoorse** [ValveSoftware/steam-runtime#457]
#### Your system information

* Steam Runtime Version: 
* Distribution (e.g. Ubuntu 18.04): Arch Linux
* Link to your full system information (Help -> System Information) in a [Gist](https://gist.github.com/):  https://gist.github.com/DadSchoorse/96c72c2abf2fc2167c8c1564cb0851ab
  
* Have you checked for system updates?: Yes
* What compatibility tool are you using?:Proton 5.13+
* If you are using Steam Linux Runtime, or Proton 5.13 or newer: What versions are listed in `SteamLinuxRuntime_soldier/VERSIONS.txt`?
```
#Name	Version		Runtime	Runtime_Version	Comment
depot	0.20210818.24			# Overall version number
pressure-vessel	0.20210809.1			
scripts	v0.20210818.0-0-gc6be089			# Entry point scripts, etc.
soldier	0.20210817.0	soldier	0.20210817.0	# soldier_platform_0.20210817.0/
```

#### Please describe your issue in as much detail as possible:
The Vulkan Validation Layers never get loaded with Proton + pressure-vessel, other layers e.g. vkBasalt work.

[slr-app4700-t20210831T150711.log](https://github.com/ValveSoftware/steam-runtime/files/7084114/slr-app4700-t20210831T150711.log)
> `15:07:11.777792: pressure-vessel-wrap[26864]: I: Failed to load Vulkan explicit layer #4 from /usr/share/vulkan/explicit_layer.d/VkLayer_khronos_validation.json: Vulkan layer file_format_version "1.2.0" in "/usr/share/vulkan/explicit_layer.d/VkLayer_khronos_validation.json" is not supported`
#### Steps for reproducing this issue:

1. Set the launch options of any proton game that uses dxvk or vkd3d-proton to `VK_INSTANCE_LAYERS=VK_LAYER_KHRONOS_validation PROTON_LOG=1 %command%`
2. Launch the game
3. Look at the log, no validation output
4. Disable pressure-vessel
5. Launch again
6. Look at the log, lot's of validation output

**Comment by smcv** [comment]
pressure-vessel doesn't know that Vulkan layer format 1.2.0 didn't introduce a new setting that is mandatory-to-understand.

From https://github.com/KhronosGroup/Vulkan-Loader/pull/605 it looks as though we should be able to just ignore all the new keys.

**Comment by smcv** [comment]
If you're comfortable with using an unofficial build, you could try replacing `SteamLinuxRuntime_soldier/pressure-vessel/` with the result of unpacking this: https://gitlab.steamos.cloud/steamrt/steam-runtime-tools/-/jobs/60012/artifacts/raw/_build/production/pressure-vessel-bin+src.tar.gz

**Comment by DadSchoorse** [comment]
Yes, that one works.

**Comment by smcv** [comment]
This has now been released in a beta. To check which version you have, look at `SteamLinuxRuntime_soldier/VERSIONS.txt`. The version that fixes this issue is `depot 0.20210907.26` or later.

**Comment by DadSchoorse** [comment]
Everything works as expected with the beta version.
